src/Windows/Avalonia.Win32/Avalonia.Win32.csproj

```diff
@@ -4,6 +4,9 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PackageId>Avalonia.Win32</PackageId>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.SDK.Contracts" Version="10.0.18362.2005" />
```

Comment:
Does that work with Win 8.1 or is it Win10+ only?

Reply:
This is Win10+ only from what I can tell.

Reply:
> The Windows 10 WinRT API Pack enables you to add the latest Windows Runtime APIs support to your .NET Framework 4.5+ and .NET Core 3.0+ libraries and apps.

https://www.nuget.org/packages/Microsoft.Windows.SDK.Contracts/10.0.17763.1000-preview

But AFAIK the toast notifications only work in Windows 10 anyway.

Reply:
Windows 8.1 also has them

Reply:
Yeah Windows 8 and 8.1 both have support for the ToastNotification API. Not sure if we really care about Windows 8 though 😄 .

Reply:
We probably want a managed fallback that displays notifications in separate windows manually. Like IM clients of the past did. That's not in the scope of this PR though.

Reply:
Well, it's more of "do people who use Avalonia for their software care about Windows 8 and 8.1", not "if we care about it".